.github/workflows/build-rpms.yaml

```diff
@@ -0,0 +1,77 @@
+        description: what to do?
+        default: run-in-bootc-container
+        options: 
+        - build-upstream-rpms
+        - run-in-bootc-container
+jobs:
+  build:
+    strategy:
+      matrix:
+        #disabled arm runner because images for arm no available in okd payload- see USHIFT-5570
+        runners: [ubuntu-24.04]
+    name: build microshift upstream
+    runs-on: ${{ matrix.runners }}
+    steps:
+      - name: Check out microshift-upstream CI repository
+        uses: actions/checkout@v4
+
+      - name: Check out microshift repository
+        uses: actions/checkout@v4
+        with:
+            repository: openshift/microshift
+            path: microshift
+            ref: main      
+
+      - name: Add cwd to path for kubectl.
+        run: echo `pwd` >> $GITHUB_PATH
+
+      - name: Build Microshift RPMs and Container
+        shell: bash
+        run: |
+          # /dev/sdb1  is mounted as /mnt - disk space needed for the podman build containers layers.
+          sudo mkdir -p /mnt/tmp/output
+          cd ${GITHUB_WORKSPACE}/
+          sudo apt-get install podman -y
+          export OKD_REPO=quay.io/okd/scos-release
+          # podman version applies bad ulimits default causing "too many open files" errors so we override ulimits.
+          TMPDIR=/mnt/tmp sudo podman build --layers=false --ulimit nofile=65536:65536 \
+                                       --build-arg OKD_VERSION_TAG=${{ inputs.okd-version }} \
+                                       --build-arg OKD_REPO=${OKD_REPO} \
+                                       --env WITH_TOPOLVM=1 \
+                                       --env WITH_FLANNEL=1 \
+                                       -f microshift-okd-multi-build.Containerfile \
+                                       -t microshift-okd \
+                                       -v /mnt/tmp/output:/output:z .
+      - name: run test verifying that Microshift is running 
+        if:  inputs.action == 'run-in-bootc-container'
+        shell: bash
+        run: |
+            sudo apt-get -y install lvm2
+            sudo truncate --size=20G /tmp/lvmdisk
+            sudo losetup -f /tmp/lvmdisk
+            device_name=$(losetup -j /tmp/lvmdisk | cut -d: -f1)
+            sudo vgcreate -f -y myvg1 ${device_name}
+            sudo lvcreate -T myvg1/thinpool -L 6G
+            sudo podman run --privileged --rm --name microshift-okd --volume /dev:/dev:rslave --hostname 127.0.0.1.nip.io -d microshift-okd 
+            sleep 3
+            # wait until everything is running
+            sudo podman exec -ti microshift-okd bash -c 'microshift healthcheck --namespace topolvm-system --deployments topolvm-controller'
+
+      - name: Release if the previous test passed
+        uses: softprops/action-gh-release@v2
+        with:
+          tag_name: test-v2
+          files: |
+            output/_output/rpmbuild/RPMS/x86_64/*.rpm
+            output/_output/rpmbuild/RPMS/noarch/*.rpm            
```

Comment:
I think we should only publish the following RPMs upstream:
* microshift + release-info
* microshift-greenboot
* microshift-networking
* microshift-selinux

We also need the following upstream-specific:
* microshift-flannel + release-info
* microshift-topolvm + release-info